settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+<pandora>
+    <!-- GLOBAL SETTINGS -->
+    <IsMonitoringEnabled>true</IsMonitoringEnabled>
+    <ShouldDisplayAlgorithmInfo>false</ShouldDisplayAlgorithmInfo>
+    <SingleHitTypeClusteringMode>true</SingleHitTypeClusteringMode>
+
+    <!-- ALGORITHM SETTINGS -->
+    <algorithm type = "LArPreProcessing">
+        <OutputCaloHitListNameU>CaloHitListU</OutputCaloHitListNameU>
+        <OutputCaloHitListNameV>CaloHitListV</OutputCaloHitListNameV>
+        <OutputCaloHitListNameW>CaloHitListW</OutputCaloHitListNameW>
+        <FilteredCaloHitListName>CaloHitList2D</FilteredCaloHitListName>
+        <CurrentCaloHitListReplacement>CaloHitList2D</CurrentCaloHitListReplacement>
+    </algorithm>
+
+    <!-- TwoDReconstruction -->
+    <algorithm type = "LArDLHitTrackShowerId">
+        <CaloHitListNames>CaloHitListU CaloHitListV CaloHitListW</CaloHitListNames>
+        <ModelFileNameU>PandoraNetworkData/PandoraUnet_TSID_DUNEFD_U_v03_25_00.pt</ModelFileNameU>
+        <ModelFileNameV>PandoraNetworkData/PandoraUnet_TSID_DUNEFD_V_v03_25_00.pt</ModelFileNameV>
+        <ModelFileNameW>PandoraNetworkData/PandoraUnet_TSID_DUNEFD_W_v03_25_00.pt</ModelFileNameW>
+    </algorithm>
```

Comment:
Pragmatic thought: Is it worth exposing the visualise option here? Not in terms of a minimal file, but something people might want on/off?